src/octoprint/plugins/logging/static/js/logging.js

```diff
@@ -21,13 +21,6 @@ $(function () {
             return _.sortBy(self.availableLoggers());
         });
 
-        self.configuredLoggersSorted = ko.computed(function () {
-            return _.sortBy(self.configuredLoggers(), function (o) {
-                o.level();
```

Comment:
Was there any point to calling `o.level()` here?  I think that calling the knockout js element without using it has no effect.

As far as I can tell, it was added in 986591aed4c1771fe4992dae19357c83bb11e731 which was a squash, so the actual addition is:

https://github.com/OctoPrint/OctoPrint/pull/2091/commits/481aa6727a8e6f829b0c61f235f65892e15849cd#diff-ee9dc3299c3eba315a9e40fa02055530ea1aa207ac30a2a9d4cb2b6522652a0dR17

There doesn't seem to be any reason for it and the code works fine without.

Reply:
Probably something only @kantlivelong can answer 😅

Reply:
Not entirely sure either really. Likely just left it in by mistake. Good find :+1: